src/buffer/out/Row.cpp

```diff
@@ -202,10 +202,15 @@ OutputCellIterator ROW::WriteCells(OutputCellIterator it, const size_t index, co
                 ++it;
             }
 
-            // If we're asked to set the wrap status and we just filled the last column with some text, set wrap status on the row.
-            if (setWrap && fillingLastColumn)
+            // If we're asked to (un)set the wrap status and we just filled the last column with some text...
+            // NOTE:
+            //  - wrap = std::nullopt    --> we don't care about changing the value

```

Comment:
FYI, I figured out when this is applicable (at least one case).

It's when we use the `Popup` class to show an overlay for CMD (like listing out the most recently used commands). We don't want the wrap status messed with during either the backup or restore. 

I think I verbally said that I wasn't sure of a circumstance where the `std::nullopt` case would be used. But now I am.

Reply:
In that case, I'm specifically changing the comment to "don't change the wrap value" for clarity.